### DISCUSSION TITLE
AudioGridder vs Transform Engine

### DISCUSSION THREAD

**Discussion by thewesleydavis2** [apohl79/audiogridder#1294] [Ideas]
I would love to see AudioGridder take on some features like https://fourieraudio.com/transform ($10K). I know there are some fundamental differences but I feel like this project is very similar and I think there is great potential in the live sound market.

**Comment by Karl-Franz** [comment]
What features in particular? Doesn't seem to be a lot of info in their website about how it actually works.

**Comment by vesalaasanen** [comment]
Fourier transform is a computer server that runs VST plugins in a live music environment, similar to Waves SuperRack LiveBox. The idea is to have a server that runs plugin chains and connects to the console via Dante, MADI, or some other digital I/O. These boxes then have a remote UI you run on a separate laptop. So, instead of just using the internal processing of the mixing console, you extend the processing to the server with plugins. You can do all this with an ordinary computer and one of the many plugin hosts, but the caveat is that you usually end up having higher latency and no sandboxing. So, if some plugin crashes, your whole host will be down.

AudioGridder has an excellent basis for becoming something like this, mainly because sandboxing is already in the engine, which is crucial for system stability. However, there is a need for at least:
- Local I/O support instead of streaming the audio
- UI to:
  - Build plugin chains
  - Patch plugin chains to local I/O
 
 I hope that there is interest in looking into this application since the options in the market carry a very high price tag.

**Reply by thewesleydavis2** [reply]
Yes, this! Sorry I never responded.

Sandboxing is a major need that most VST hosts don't have.
